services/oodikone2-frontend/src/components/PopulationSearchForm/index.jsx

```diff
@@ -612,7 +641,7 @@ class PopulationSearchForm extends Component {
 
         <Message error color="blue" header={errorText} />
 
-        <Form.Button onClick={this.fetchPopulation} disabled={isQueryInvalid}>
+        <Form.Button onClick={() => this.fetchPopulation(query)} disabled={isQueryInvalid}>
```

Comment:
In my opinion we should use function calls like this less and more `history.push()` or similar.
When not using URL we have bugs where the URL is different to the actual page content.

The browser back button does not work.
Page after filling the form, pressing the button and then pressing the X button to go back to the form:
![image](https://user-images.githubusercontent.com/468816/57549976-a34ef100-736d-11e9-8cb1-3937d446bd27.png)